rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -690,7 +690,7 @@ public int scoringAddressesPunishmentIncrement() {
 
     public long scoringAddressesPunishmentMaximumDuration() {
         // default value: 1 week
-        return getLong("scoring.addresses.maximum", 60 * 24 * 7)  * 60000L;
+        return getLong("scoring.addresses.maximum", 60L * 24 * 7)  * 60000L;
```

Comment:
Is it possible to use methods that represent these conversions? I think we could use something like:

```java
return TimeUnit.MINUTES.toMillis(getLong("scoring.addresses.maximum", TimeUnit.DAYS.toMinutes(7)));
```

I don't think it's the best, but it does make clear that:

1. `"scoring.addresses.maximum"` is a duration in minutes
1. `scoringAddressesPunishmentMaximumDuration` returns a duration in millis